Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1112,16 +1112,19 @@ CGAffineTransform CGContextGetTextMatrix(CGContextRef context) {
 */
 void CGContextSetTextPosition(CGContextRef context, CGFloat x, CGFloat y) {
     NOISY_RETURN_IF_NULL(context);
-    UNIMPLEMENTED();
+    context->textMatrix.tx = x;
```

Comment:
nit: can we use a inline function to take both of them in and set.

context->setTextMatrix(x,y);